docs/flexbox.md

```diff
@@ -63,13 +106,29 @@ export default class JustifyContentBasics extends Component {
 // skip this line if using Create React Native App
 AppRegistry.registerComponent('AwesomeProject', () => JustifyContentBasics);
 ```
+See some examples of [wix](https://medium.com/wix-engineering/the-full-react-native-layout-cheat-sheet-a4147802405c) engineers
```

Comment:
I think we should consider adding a `"Source: Wix Engineers"` with a link below the images instead of each one having a leading sentence